Description: Initialize kernel version, to cope with empty sublevel definitions
 Example:
 .
 VERSION = 3
 PATCHLEVEL = 3
 SUBLEVEL =
 EXTRAVERSION =
 NAME = Saber-toothed Squirrel
Signed-off-by: Stefan Lippers-Hollmann <s.l-h@gmx.de>

--- a/src/prog/gpm-root.y
+++ b/src/prog/gpm-root.y
@@ -1161,7 +1161,7 @@ int main(int argc, char **argv)
    */
    {
       struct utsname linux_info;
-      int v1,v2,v3;
+      int v1=0,v2=0,v3=0;
       struct stat sbuf;
 
       if (uname(&linux_info)) {
